lib/honeybadger/splitproject.rb

```diff
@@ -1,10 +1,22 @@
 require 'honeybadger/honeybadger'
 require 'honeybadger/key_helpers'
+require 'honeybadger/team_filter'
 
 module Honeybadger
   module Splitproject
-    
+    @@honeybadger_filters = [TeamFilter.new]
+
     class << self
+
+      def filters
+        @@honeybadger_filters
+      end
+
+      def add_filter(klass)
+        raise "MissingFilterMethod" unless klass.respond_to?(:filter)
```

Comment:
Not sure I like this construct. What if the class has a ```filter``` method that takes different args and does something totally different? Seems like we should just have a superclass called HoneybadgerFilter or something, and check ```klass.is_a?```, and then have TeamFilter extend HoneybadgerFilter.